docs/proposals/YYYYMMDD-template.md

```diff
@@ -104,7 +69,8 @@ A table of contents is helpful for quickly jumping to sections of a proposal and
   - [Alternatives [optional]](#alternatives-optional)
   - [Infrastructure Needed [optional]](#infrastructure-needed-optional)
 
-[Tools for generating]: https://github.com/ekalinin/github-markdown-toc
+## Glossary
+Refer to the [Cluster API Book Glossary](https://cluster-api.sigs.k8s.io/reference/glossary.html).
```

Comment:
If we are going to have a Glossary section, shouldn't it refer to new items defined here that wouldn't likely be applicable to already be in the book?

Reply:
I'd expect new terms to be added as part of the PR to the glossary

Reply:
Added a comment to make it clear